example/figure_eight_plan_6dof.py

```diff
@@ -0,0 +1,172 @@
+# Python standard lib
+import os
+import sys
+import pathlib
+
+# ROS
+# import rospy
+# from sensor_msgs.msg import JointState
+
+# PyBullet
+import pybullet_api
+
+# OpTaS
+import optas
+
+
+class Planner:
+
+
+    def __init__(self):
+
+        cwd = pathlib.Path(__file__).parent.resolve() # path to current working directory
+        pi = optas.np.pi  # 3.141...
+        self.T = 50 # no. time steps in trajectory
+        link_ee = 'end_effector_ball'  # end-effector link name
+        self.Tmax = 10.  # trajectory of 5 secs
+        t = optas.linspace(0, self.Tmax, self.T)
+        self.dt = float((t[1] - t[0]).toarray()[0, 0])  # time step
+
+        # Setup robot
+        urdf_filename = os.path.join(cwd, 'robots', 'kuka_lwr.urdf')
+        self.kuka = optas.RobotModel(
+            urdf_filename=urdf_filename,
+            time_derivs=[0, 1],  # i.e. joint position/velocity trajectory
+            param_joints='lwr_arm_0_joint',
```

Comment:
This will likely throw an error somewhere down the line. `param_joints` is expected to be a list, so this line should be
```python
param_joints = ['lwr_arm_0_joint']
```